jwst/lib/set_telescope_pointing.py

```diff
@@ -285,21 +287,34 @@ def __str__(self):
 R2A = 3600.0 * R2D
 PI2 = np.pi * 2.0
 
-# Pointing container
-# Attributes are as follows. Except for the observation time, all values
-# are retrieved from the engineering data.
-#    q            : Quaternion of the FGS.
-#    j2fgs_matrix : J-frame to FGS transformation.
-#    fsmcorr      : Fine Steering Mirror position.
-#    obstime      : Mid-point time of the observation at which all other values have been
-#                   calculated.
-#    gs_commanded : Guide star position as originally commanded.
-#    fgsid        : FGS in use, 1 or 2.
-#    gs_position  : X/Y guide star position in the FGS.
 Pointing = namedtuple(
     "Pointing", ["q", "j2fgs_matrix", "fsmcorr", "obstime", "gs_commanded", "fgsid", "gs_position"]
 )
 Pointing.__new__.__defaults__ = (None,) * 5
+"""
+Pointing container.
```

Comment:
This should have worked but it is not showing under https://jwst-pipeline--9907.org.readthedocs.build/en/9907/api/jwst.lib.set_telescope_pointing.Pointing.html#jwst.lib.set_telescope_pointing.Pointing . How badly do you want this rendered? Is whatever showing there enough? If not, I can investigate. This is currently not even showing on `main`. 🤷‍♀️

Reply:
Ah, good catch. Done. Thanks!